blueman/gui/manager/ManagerDeviceMenu.py

```diff
@@ -213,6 +224,18 @@ def Generate(self):
             self.append(item)
             return
 
+        # Generic (dis)connect
+        if not row["connected"]:
+            connect_item = create_menuitem(_("_Connect"), get_icon("blueman", 16))
```

Comment:
Profiles that are connected in the generic connect method have the `auto_connect` attribute set in their `btd_profile` struct. Those are A2DP source, A2DP sink, and HID (It looks like our HandsfreeGateway service did not really make sense) . Another improvement would be to not show the generic connect item in case none of them are available.